treeherder/model/derived/jobs.py

```diff
@@ -128,7 +127,6 @@ class JobsModel(TreeherderModelBase):
     ]
 
     LOWER_TIERS = [2, 3]
```

Comment:
This could be deleted if you reference the value from the ``ExclusionProfileManager``.